frappe/__init__.py

```diff
@@ -188,14 +188,19 @@ def connect(site=None, db_name=None):
 	local.db = Database(user=db_name or local.conf.db_name)
 	set_user("Administrator")
 
-def connect_read_only():
+def connect_secondary():
```

Comment:
`replica` instead of `secondary` ?

From wikipedia

<img width="955" alt="Screenshot 2019-05-10 at 8 33 29 AM" src="https://user-images.githubusercontent.com/140076/57499711-5122b700-72fe-11e9-9046-68787b89eab3.png">